#goal - joint angles for various joints of the arm
float64 shoulder_pan
float64 shoulder_tilt
float64 elbow_tilt
float64 wrist_rotate
---
#result - success result & final joints position [shoulder pan, shoulder tilt, elbow tilt, wrist rotate]
bool success
float64[] arm_position
---
#result - current joints position [shoulder pan, shoulder tilt, elbow tilt, wrist rotate]
float64[] arm_position
